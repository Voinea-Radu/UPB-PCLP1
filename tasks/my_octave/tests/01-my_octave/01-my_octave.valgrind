==407995== Memcheck, a memory error detector
==407995== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==407995== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==407995== Command: /root/CLionProjects/pclp/tema1/my_octave
==407995== Parent PID: 407994
==407995== 
==407995== 
==407995== HEAP SUMMARY:
==407995==     in use at exit: 0 bytes in 0 blocks
==407995==   total heap usage: 24 allocs, 24 frees, 8,624 bytes allocated
==407995== 
==407995== All heap blocks were freed -- no leaks are possible
==407995== 
==407995== For lists of detected and suppressed errors, rerun with: -s
==407995== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
