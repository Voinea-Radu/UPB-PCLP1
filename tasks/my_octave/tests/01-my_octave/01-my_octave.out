No data with the given index
2 6
7 7 -1 -1 -10 -6 
-9 2 2 7 6 2 
88 -11 -19 -64 -64 -24 
133 56 -24 -59 -142 -74 
-109 -10 22 67 94 42 
-46 -24 8 18 52 28 
-86 2 18 58 68 28 
